### PR TITLE
customize welcomebot for falcon

### PR DESCRIPTION
**Pull request by fboxwala**
falconbot is alive and is cute

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -187,9 +187,9 @@ def bot_hello(greeting, actor, ircsock, channel):
 
 # Explains what the bot is when queried.
 def bot_help(ircsock, channel):
-    ircsock.send("PRIVMSG {} :I'm a bot!  I'm from here <https://github"
-                 ".com/shaunagm/oh-irc-bot>.  You can change my behavior by "
-                 "submitting a pull request or by talking to shauna"
+    ircsock.send("PRIVMSG {} :I'm a bot!  I'm a fork of shauna's welcomebot,"
+		 " you can checkout my internals and contrinute here: "
```
Comment (kgriffs) [inline bot.py]: Typo: contrinute

**Comment by fboxwala** [comment]
fixed typo!

File: bot_settings.py [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 wait_time = 60
-channel_greeters = ["shauna", "paulproteus", "marktraceur"]
+channel_greeters = ["kgriffs"]
 nick_source = "nicks.csv"
 
 # Bot text
 hello_list = ["hello", "hi", "hey", "yo", "sup"]
 help_list = ["help", "info", "faq", "explain_yourself"]
-welcome_message = "Welcome! The channel is pretty quiet right now, so I thought I'd say hello, and ping some people that you're here. If no one responds for a whle, try emailing us at hello@openhatch.org or just try coming back later. FYI, you're now on my list of known nicknames, so I won't bother you again."
+welcome_message = "Welcome! The channel is pretty quiet right now, so I thought I'd say hello"", and ping some people that you're here (like kgriffs). I'm a bot! If no one responds for"
```
Comment (kgriffs) [inline bot_settings.py]: Line continuations. :) http://docs.python-guide.org/en/latest/writing/style/#line-continuations

File: bot_settings.py [hunk 3]
```diff
@@ -1,17 +1,20 @@
...
 wait_time = 60
-channel_greeters = ["shauna", "paulproteus", "marktraceur"]
+channel_greeters = ["kgriffs"]
 nick_source = "nicks.csv"
 
 # Bot text
 hello_list = ["hello", "hi", "hey", "yo", "sup"]
 help_list = ["help", "info", "faq", "explain_yourself"]
-welcome_message = "Welcome! The channel is pretty quiet right now, so I thought I'd say hello, and ping some people that you're here. If no one responds for a whle, try emailing us at hello@openhatch.org or just try coming back later. FYI, you're now on my list of known nicknames, so I won't bother you again."
+welcome_message = "Welcome! The channel is pretty quiet right now, so I thought I'd "
```
Comment (kgriffs) [inline bot_settings.py]: You'll need to wrap in parens to get the strings to concatenate, e.g.:

```py
welcome_message = ("Welcome! The channel..."
                   "say hello, and ping some....")
```

File: bot_settings.py [hunk 4]
```diff
@@ -1,17 +1,20 @@
...
 wait_time = 60
-channel_greeters = ["shauna", "paulproteus", "marktraceur"]
+channel_greeters = ["kgriffs"]
 nick_source = "nicks.csv"
 
 # Bot text
 hello_list = ["hello", "hi", "hey", "yo", "sup"]
 help_list = ["help", "info", "faq", "explain_yourself"]
-welcome_message = "Welcome! The channel is pretty quiet right now, so I thought I'd say hello, and ping some people that you're here. If no one responds for a whle, try emailing us at hello@openhatch.org or just try coming back later. FYI, you're now on my list of known nicknames, so I won't bother you again."
+welcome_message = ("Welcome! The channel is pretty quiet right now, so I thought I'd "
```
Comment (kgriffs) [inline bot_settings.py]: Almost perfect! One last thing. In Python it's considered good style to indent in one of two ways:

```py
long_string = ('first line'
               'second line')
```

```py
long_string = (
    'first line'
    'second line'
)
```

File: bot_settings.py [hunk 5]
```diff
@@ -1,17 +1,20 @@
...
-botnick = "welcomebot"
+channel = "#falconframework"
+botnick = "Elaenor"
 server = "irc.freenode.net"
 registered = False
 
 # Bot behavior
 wait_time = 60
-channel_greeters = ["shauna", "paulproteus", "marktraceur"]
+channel_greeters = ["kgriffs"]
```
Comment (kgriffs) [inline bot_settings.py]: While we're at it, can we add `jvrbanac`? Thanks!

File: bot_settings.py [hunk 6]
```diff
@@ -1,17 +1,20 @@
...
 nick_source = "nicks.csv"
 
 # Bot text
 hello_list = ["hello", "hi", "hey", "yo", "sup"]
 help_list = ["help", "info", "faq", "explain_yourself"]
-welcome_message = "Welcome! The channel is pretty quiet right now, so I thought I'd say hello, and ping some people that you're here. If no one responds for a whle, try emailing us at hello@openhatch.org or just try coming back later. FYI, you're now on my list of known nicknames, so I won't bother you again."
+welcome_message = ("Welcome! The channel is pretty quiet right now, so I thought I'd "
+"say hello, and ping some people that you're here (like kgriffs). I'm a bot! If no one "
```
Comment (kgriffs) [inline bot_settings.py]: See note above about `jvrbanac`.

File: bot_settings.py [hunk 7]
```diff
@@ -1,17 +1,24 @@
...
 nick_source = "nicks.csv"
 
 # Bot text
 hello_list = ["hello", "hi", "hey", "yo", "sup"]
 help_list = ["help", "info", "faq", "explain_yourself"]
-welcome_message = "Welcome! The channel is pretty quiet right now, so I thought I'd say hello, and ping some people that you're here. If no one responds for a whle, try emailing us at hello@openhatch.org or just try coming back later. FYI, you're now on my list of known nicknames, so I won't bother you again."
+welcome_message = ("Welcome! The channel is pretty quiet "
+                   "right now, so I thought I'd say "
+                   "hello, and ping some people that "
+                   "you're here (like kgriffs, jvrbanac).
```
Comment (kgriffs) [inline bot_settings.py]: [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself): Is there a way to use channel_greeters here?

**Comment by kgriffs** [comment]
Just one more suggestion (inline), then this is ready to merge. Thanks!

**Comment by kgriffs** [comment]
LGTM, thanks! Has this been deployed?

**Comment by fboxwala** [comment]
it will as soon as I remember my password for that box :P

**Pull request merged**